starter_code/queries.md

```diff
@@ -5,79 +5,105 @@
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
 <!-- Your Code Goes Here -->
+filter: {name: "Babelgum"}
+projection: {name: 1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
 <!-- Your Code Goes Here -->
+filter: {number_of_employees: {$gt: 5000}}
+sort: {number_of_employees: 1}
+
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
+filter: {founded_year: {$gt: 2000}, founded_year: {$lt:2005}}
+projection: {name: 1, founded_year: 1}
```

Comment:
Your filter should include the years 2005 and 2000, so it should be $lte and $gte